ci-operator/config/openshift/aws-load-balancer-controller/openshift-aws-load-balancer-controller-openshift-rel-1.0.yaml

```diff
@@ -0,0 +1,42 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+binary_build_commands: make controller
+build_root:
+  from_repository: true
+canonical_go_repository: sigs.k8s.io/aws-load-balancer-controller
+images:
+- dockerfile_path: Dockerfile.openshift
+  from: base
+  to: aws-load-balancer-controller
+promotion:
+  namespace: aws-load-balancer-operator
+  tag: "1.0"
+releases:
+  initial:
+    integration:
+      name: "4.12"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.12"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: go test -race ./pkg/... ./webhooks/...
+  container:
+    from: src
+zz_generated_metadata:
+  branch: openshift-rel-1.0
```

Comment:
Why did external DNS not have this issue? Could you just call it `release-1.0` like External DNS?
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/external-dns/openshift-external-dns-release-1.0.yaml

Reply:
`release-x.y` pattern is used upstream in `kubernetes-sigs` org. I had to think of another pattern to avoid a clash.

Reply:
Are you talking about the branches in kubernetes-sigs/aws-load-balancer-controller: https://github.com/kubernetes-sigs/aws-load-balancer-controller/branches?